src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -800,7 +800,7 @@ public void submoduleClean(boolean recursive) throws GitException, InterruptedEx
     	if (recursive) {
             args.add("--recursive");
     	}
-    	args.add("git clean -fdx");
+        args.add("git clean -ffdx");
```

Comment:
Sure, no problem. There's a mix of tabs and spaces used for indenting in several places, but perhaps that should be cleaned up in a separate commit.